helm_charts_updater/git.py

```
@@ -1,7 +1,16 @@
+"""Git repository operations for helm-charts-updater.
+
+This module provides functionality to clone, commit, and push changes
+to a GitHub repository containing Helm charts.
+"""
+
 import logging
 import os
+import re
 import sys
 from pathlib import Path
+from typing import List
```

Ruff (0.14.13)

`typing.List` is deprecated, use `list` instead

(UP035)